examples/kubernetes_orchestration/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+
+pre_run () {
+  zenml integration install sklearn facets kubernetes
```

Comment:
I couldn't find any place where facets was actually used. If I missed it, I personally try to keep the examples pretty light-weight and non-overlapping with other integrations to just showcase one feature, but not sure what others think about that.
```suggestion
  zenml integration install sklearn kubernetes
```

Reply:
See discussion on skew_comparison step below

Reply:
Not removing it since we will change the example to use facets in the post-execution